CMakeLists.txt

```diff
@@ -753,8 +753,20 @@ if(EXECUTORCH_BUILD_PYBIND)
       PROPERTIES # Assume that this library will be installed in
                  # `site-packages/executorch/extension/pybindings`, and that
                  # the torch libs are in `site-packages/torch/lib`.
-                 BUILD_RPATH "@loader_path/../../../torch/lib"
-                 INSTALL_RPATH "@loader_path/../../../torch/lib"
+                 BUILD_RPATH "@loader_path/../../../torch/lib" "@loader_path/../../" "@loader_path/../../examples/models/llama2/custom_ops"
```

Comment:
While you're in here, please add a TODO like "TODO: Install the custom_ops libs into the same directory as portable_lib." I really don't like making our library point under a directory called "examples".